server/src/controllers/temp-outfitter.es6

```diff
@@ -327,6 +365,21 @@ const getAllFileNames = applicationId => {
   });
 };
 
+tempOutfitter.updateApplicationModel = (model, submitted, user) => {
+  if (user.role === 'admin') {
```

Comment:
I'm unclear here what the admin has privilege to do on this front?

Reply:
Only an admin can change the `status` to `Accepted` or update the `applicantMessage`